client/src/component/CohortTrainees.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+
+export const TraineesTable = () => {
+  const [name, setName] = useState("");
+  const [trainees, setTrainees] = useState([]);
+
+  const handleNameChange = (event) => {
+    setName(event.target.value);
+  };
+
+  const handleAddTrainee = (event) => {
+    event.preventDefault(); // Prevent of loading page
+    if (name.trim() !== "") {
+      setTrainees([...trainees, name]);
+      setName("");
+    }
+  };
+
+  const handleDeleteTrainee = (index) => {
+    const updatedTrainees = [...trainees];
+    updatedTrainees.splice(index, 1);
+    setTrainees(updatedTrainees);
+  };
```

Comment:
This is just removing the user locally which could make users incorrectly think that the functionality has been implemented. You could just alert something from this function so that it is clear that it is not working code